Backend/webblog/accounts/views.py

```diff
@@ -30,7 +30,7 @@ def post(self, request, *args, **kwargs):
         user = serializer.validated_data
         return Response({
             "user": UserSerializer(user, context=self.get_serializer_context()).data,
-            "token": AuthToken.objects.create(user)[1]
+            "token": AuthToken.objects.create(user, )[1]
```

Comment:
Why this comma added while creating AuthToken across multiple files ?

Reply:
might've been added by the Pycharm.
it's been removed now.